src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -1873,62 +1873,62 @@ f1_elc_oper_rev_nb,1,1994,Includes $ unbilled revenues & MWH relating to unbille
 f1_elc_oper_rev_nb,1,1995,"Line 12, Col (b) inc $ unbill rev, Col (d) inc MWH relating unbill rev",,,
 f1_elctrc_erg_acct,1,1994,SOURCES OF ENERGY,header,#N/A,
 f1_elctrc_erg_acct,2,1994,Generation (Excluding Station Use):,header,#N/A,
-f1_elctrc_erg_acct,3,1994,Steam,,steam_generation,
-f1_elctrc_erg_acct,4,1994,Nuclear,,nuclear_generation,
-f1_elctrc_erg_acct,5,1994,Hydro-Conventional,,hydro_conventional_generation,
-f1_elctrc_erg_acct,6,1994,Hydro-Pumped Storage,,hydro_pumped_storage_generation,
-f1_elctrc_erg_acct,7,1994,Other,,other_energy_generation,
-f1_elctrc_erg_acct,8,1994,Less Energy for Pumping,,pumping_energy,
-f1_elctrc_erg_acct,9,1994,Net Generation (Enter Total of lines 3 through 8),total,net_energy_generation,
-f1_elctrc_erg_acct,10,1994,Purchases,,megawatt_hours_purchased_dbfonly,"Split into 2 megawatt_hours_purchased_ fields in XBRL, which should probably be summed to produce this field."
+f1_elctrc_erg_acct,3,1994,Steam,reported_value,steam_generation,
+f1_elctrc_erg_acct,4,1994,Nuclear,reported_value,nuclear_generation,
+f1_elctrc_erg_acct,5,1994,Hydro-Conventional,reported_value,hydro_conventional_generation,
+f1_elctrc_erg_acct,6,1994,Hydro-Pumped Storage,reported_value,hydro_pumped_storage_generation,
+f1_elctrc_erg_acct,7,1994,Other,reported_value,other_energy_generation,
+f1_elctrc_erg_acct,8,1994,Less Energy for Pumping,reported_value,pumping_energy,
+f1_elctrc_erg_acct,9,1994,Net Generation (Enter Total of lines 3 through 8),calculated_value,net_energy_generation,
+f1_elctrc_erg_acct,10,1994,Purchases,reported_value,megawatt_hours_purchased_dbfonly,"Split into 2 megawatt_hours_purchased_ fields in XBRL, which should probably be summed to produce this field."
```

Comment:
i like the rename of these row types.... but does anything need to be changed downstream to accommodate that with that?

Reply:
It turned out that these values were not yet being referenced anywhere, so I didn't need to change anything downstream. For the moment they are just serving to document the table. I think they'll become more useful when we get into the aggregation processes.